mirrord/agent/src/main.rs

```diff
@@ -7,6 +7,7 @@
 #![feature(tcp_quickack)]
 #![feature(async_fn_in_trait)]
 #![allow(incomplete_features)]
+#![warn(clippy::indexing_slicing)]
```

Comment:
Isn't there a more centralized option for enabling this, like the configuration toml?

Reply:
As of now there's no way to enable lints globally via configuration files. I could just add this lint to clippy arguments in the CI, but I bet it would be annoying